src/app/places/places.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { PagesService } from '../pages.service';
+
+interface Place {
+  Id: number;
+  Name: String;
+  Address: String;
+  Phone: String;
+  Link: String;
+  Max_persons: String;
+  Tag: String;
+  Description: String;
+}
```

Comment:
in my opinion any interface should be placed in separate file in models folder

Reply:
I make a folder called models and file in this folder called place.